deltachat-ffi/deltachat.h

```diff
@@ -1974,6 +1974,33 @@ void            dc_delete_msgs               (dc_context_t* context, const uint3
 void            dc_forward_msgs              (dc_context_t* context, const uint32_t* msg_ids, int msg_cnt, uint32_t chat_id);
 
 
+/**
+ * Save a copy of messages in "Saved Messages".
+ *
+ * In contrast to forwarding messages,
+ * information as author, date and origin are preserved.
+ * The action completes locally, so "Saved Messages" do not show sending errors in case one is offline.
+ * Still, a sync message is emitted, so that other devices will save the same message,
+ * as long as not deleted before.
+ *
+ * To check if a message was saved, use dc_msg_get_saved_msg_id(),
+ * UI may show an indicator and offer an "Unsave" instead of a "Save" button then.
+ *
+ * The other way round, from inside the "Saved Messages" chat,
+ * UI may show in indicator checking dc_msg_get_original_msg_id() and dc_msg_get_original_chat_id()
+ * and offer a button to go the original chat.
```

Comment:
```suggestion
 * and offer a button to go the original chat.
 * Also, show an "unsave" instead of "save" button, and hide the "delete" button.
```

At least, this is what I would expect as a user.

Reply:
agree with unsave (it is there in the iOS implementation), but hiding the delete button is a bit unexpected, i tried that, but i do not like it - it complicates things as you can add things to "Saved Messages" also otherwise, and you will wonder, which message can be "deleted" which way. also, this would make multi-select very ugly

Reply:
i changed the comment accordingly